builder/evaluator.go

```diff
@@ -417,7 +417,7 @@ func platformSupports(command string) error {
 		return nil
 	}
 	switch command {
-	case "expose", "volume", "user", "stopsignal":
+	case "expose", "volume", "user", "stopsignal", "arg":
```

Comment:
@jhowardmsft why is `arg` checked both here and in the dispatcher? one of them should be sufficient no?